pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -162,7 +162,7 @@ func (f *Fit) Filter(ctx context.Context, cycleState *framework.CycleState, pod
 	// We will keep all failure reasons.
 	var failureReasons []string
```

Comment:
should we allocate for `len(insufficientResources)` as well?

Reply:
make sense, done.